crates/chia-sdk-driver/src/primitives/datalayer/datastore_info.rs

```diff
@@ -125,6 +127,7 @@ impl<N, D: ClvmDecoder<Node = N>> FromClvm<D> for DataStoreMetadata {
                 "l" => metadata.label = Some(String::from_clvm(decoder, ptr)?),
                 "d" => metadata.description = Some(String::from_clvm(decoder, ptr)?),
                 "b" => metadata.bytes = Some(u64::from_clvm(decoder, ptr)?),
+                "p" => metadata.size_proof = Some(Bytes32::from_clvm(decoder, ptr)?),
```

Comment:
easy to mistake with CATalog precision; I would suggest `sp`

Reply:
fixed